File: src/main/java/net/minheur/potoflux/utils/ressourcelocation/ResourceLocationException.java
package net.minheur.potoflux.utils.ressourcelocation;

/**
 * Thrown when there's an error in a ResourceLocation
 */
public class ResourceLocationException extends RuntimeException {
    /**
     * Constructs a new ResourceLocationException.
     */
    public ResourceLocationException(String message) {
        super(message);
    }

    /**
     * Constructs a new ResourceLocationException.
     */
    public ResourceLocationException(String message, Throwable cause) {
        super(message, cause);
    }
}

File: src/main/kotlin/net/minheur/potoflux/utils/ressourcelocation/ResourceLocationException.kt
package net.minheur.potoflux.utils.ressourcelocation

/**
 * Thrown when there's an error in a ResourceLocation
 */
class ResourceLocationException : RuntimeException {
    constructor(message: String) : super(message)
    constructor(message: String, cause: Throwable) : super(message, cause)
}
